PatternFinding.py

```diff
@@ -81,25 +83,29 @@ def FindingQRPatterns(self, nooflevels):
                 x, y, w, h = cv.boundingRect(self.Contours[mapping])
                 cv.rectangle(
                     self.image, (x, y), (x + w, y + h), (0, 255, 0), 2)
-                cv.imshow('hello', self.image)
+                #cv.imshow('contour of qr', self.image)         #uncomment to debug
+                #cv.waitKey(0)
                 QRPatterns.append(patterns[ind])
+            #cv.destroyAllWindows()
```

Comment:
indentation?

Reply:
I think all indentations are correct. Did you encounter any error?